packages/graphback/src/buildGraphbackAPI.ts

```diff
@@ -0,0 +1,106 @@
+import { GraphQLSchema } from 'graphql';
+import { GraphbackPlugin, GraphbackPluginEngine, GraphbackCRUDGeneratorConfig, printSchemaWithDirectives, ModelDefinition } from '@graphback/core';
+import { SchemaCRUDPlugin } from '@graphback/codegen-schema';
+import { LayeredRuntimeResolverCreator, GraphbackCRUDService, DataProviderCreator, createCRUDService } from '@graphback/runtime';
+import { PubSub } from 'graphql-subscriptions';
+
+export interface GraphbackAPIConfig {
+  /**
+   * Function which creates a default CRUD Service for every data model
+   */
+  serviceCreator?: Function
+  /**
+   * Function which creates a default data provicer for every data model
+   */
+  dataProviderCreator: Function
+  /**
+   * Global CRUD configuration
+   */
+  crud?: GraphbackCRUDGeneratorConfig
+  /**
+   * Schema plugins to perform automatic changes to the schema
+   */
+  plugins?: GraphbackPlugin[]
+}
+
+interface GraphbackServices {
+  [modelName: string]: GraphbackCRUDService
+}
+
+export interface GraphbackAPI {
```

Comment:
Comment missing

Reply:
> Would you see me as ashole to ask to made some docs updates in this PR. Happy to have separate if needed.
> 
> Sorry for pointing some obvious docs stuff. Just making sure that we have the same vision of docs

I wanted to get the code for review and I can work on docs while ye review (quicker turnaround)